pydis_site/apps/forms/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class FormsConfig(AppConfig):
+    """Django AppConfig for the forms app."""
+
+    name = "pydis_site.apps.forms"
```

Comment:
Is this supposed to match the module name? Can we avoid hardcoding the name? Maybe something with `importlib` can help?